static/css/photon-site/footer.less

```diff
@@ -84,6 +84,7 @@
 
 .Footer-links-header {
     margin: 40px 0 0;
+    font-weight: 700;
```

Comment:
is the `font-weight: 700` a few lines below inside `a:link, a:visited` obsolete now?  (You'll have to try it out as css priorities are weird)

Reply:
@eviljeff Thats not obsolete because thats for the headers with some link for example the first section header Addons is linked to addons.mozilla.org, but Browsers and Products are not linked to anything so need to add font-weight: 700 explicitely

Reply:
yes, I understand why we need the new statement under `.Footer-links-header`, I was just wondering if the statement below was obsolete because `a:link, a:visited` inherits from `.Footer-links-header`.  It's not worth bothering about though.